vuejs/src/org/jetbrains/vuejs/intentions/expandVModel/VueExpandVModelIntention.kt

```diff
@@ -0,0 +1,70 @@
+package org.jetbrains.vuejs.intentions.expandVModel
+
+import com.intellij.lang.javascript.intentions.JavaScriptIntention
+import com.intellij.openapi.application.WriteAction
+import com.intellij.openapi.command.CommandProcessor
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.xml.XmlAttribute
+import com.intellij.psi.xml.XmlElementType
+import org.jetbrains.vuejs.codeInsight.tags.VueElementDescriptor
+
+class VueExpandVModelIntention : JavaScriptIntention() {
+    override fun getFamilyName(): String = "Expand v-model into :value and @input/@change"
+    override fun getText(): String = this.familyName
+    private val validModifiers = setOf("lazy", "number", "trim")
+
+    override fun isAvailable(project: Project, editor: Editor?, element: PsiElement): Boolean {
+        return element.node.elementType == XmlElementType.XML_NAME &&
+                element.parent?.node?.elementType == XmlElementType.XML_ATTRIBUTE &&
+                isValid(element)
```

Comment:
Please add a check whether you are in a Vue context - `org.jetbrains.vuejs.index.isVueContext(PsiElement)`

Reply:
I thought about that, too. Will do ;)